pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+Put description here...
+
+# Code-Reviewer Section
+The general guidlines can be found [here](https://github.com/apple/foundationdb/wiki/FoundationDB-Commit-Process).
+
+Please check each of the following things and check *all* boxes before accepting a PR.
+
+[ ] The PR has a description.
```

Comment:
"The PR has a description." --> "The PR has a description, explaining both the problem and the solution."